mev-commit-cli.sh

```diff
@@ -65,7 +65,12 @@ NEXT_PUBLIC_WALLET_CONNECT_ID=
 DD_KEY=${datadog_key}
 EOF
 
-    make -C "$GETH_POA_PATH"/geth-poa up-prod-settlement
+    # Run settlement layer
+    # Set environment variables
+    export AGENT_BASE_IMAGE=nil
```

Comment:
This should have value of `shaspitz/hyperlane-agent-mac:6985064` for spinning up settlement layer locally on mac, or `gcr.io/abacus-labs-dev/hyperlane-agent@sha256:854f92966eac6b49e5132e152cc58168ecdddc76c2d390e657b81bdaf1396af0` for amd

Reply:
Can make this a flag or param